portality/static/js/formulaic.js

```diff
@@ -1382,16 +1382,17 @@ var formulaic = {
             this.init = function() {
                 var viewClass = edges.css_classes(this.ns, "view");
                 var closeClass = edges.css_classes(this.ns, "close");
+                var viewDivClass = edges.css_classes(this.ns, "viewDiv")
+                this.divs = $("div[name='" + this.fieldDef["name"] + "__group']").find("textarea");
```

Comment:
Are we sure that replacing the `this.divs` member variable with a set of `textarea`s doesn't have any side effects in other methods that use the divs?

Reply:
You're right, it shouldn't be done this way. I will simply use a new local var.